### ISSUE TITLE
Prometheus Metrics middleware example in documentation incomplete.

### ISSUE THREAD

**Issue by gannicottb** [http4s/http4s#3278]
The documentation for the Prometheus metrics middleware found at [https://http4s.org/v0.21/middleware/](https://http4s.org/v0.21/middleware/ ) is incomplete insofar as it does not mention the final step needed to expose the metrics for scraping by a Prometheus server. The PrometheusExportService does exist, but that fact is non-obvious and requires source code spelunking to discover. It would be great if the example in the documentation included usage of PrometheusExportService, perhaps along these lines?

(from @hamnis in the Gitter):
```
def server[F[_]: Sync](api: HttpRoutes[F]): Resource[F, Server[F]] =
    for {
      registry <- Prometheus.collectorRegistry[F]
      metrics <- Prometheus.metricsOps[F](registry, "server")
      prometheus <- PrometheusExportService.build[F]
      routes = Router(
        "/api" -> Metrics[F](metrics)(api),
        "/" -> prometheus.routes
      )
      server <- BlazeServerBuilder[F]
        .bindHttp("9000", "0.0.0.0")
        .withHttpApp(routes.orNotFound)
        .resource
    } yield server
```

**Comment by gannicottb** [comment]
Working on a PR to address this now.

**Comment by rossabaker** [comment]
Closed by #3279.
